public/css/eventList.css

```diff
@@ -0,0 +1,80 @@
+.upper span {
+    display: block;
+    text-align: center;
+}
+
+.mid span {
+    display: block;
+    text-align: center;
+}
+
+#eventTitle {
```

Comment:
Style using classes, not IDs.

Reply:
but there will only ever be one of them?!

Reply:
ids should be used for js targeting, not css targeting

20f559b

Reply:
Html needs to be changed too.